README.md

```diff
@@ -3,14 +3,14 @@ GrantNav - Explore Grants in the 360 data standard
 
 [![Build Status](https://travis-ci.org/OpenDataServices/grantnav.svg?branch=master)](https://travis-ci.org/OpenDataServices/grantnav)
 
-We use HuBoard to provide an "agile board" view of our issues https://huboard.com/OpenDataServices/grantnav
+We use HuBoard to provide a Kanban board visualisation of our workflow and issues https://huboard.com/OpenDataServices/grantnav
 
 Introduction
 ------------
 
 This is a search tool for data in the 360 giving data format.
 
-This application is currently in a pre-alpha state.
+This application is currently undergoing final testing before a production launch.
```

Comment:
I was trying to avoid the word 'beta' - I don't think we gain anything from it, and everyone has different ideas of what it means. It's going live to the world this month - and it's not like any software is actually ever finished!